### PR TITLE
Add SpecFlow swag

### LINKED ISSUE #716: SpecFlow contributor package
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : SpecFlow

Q. Where have you seen the announcement (URL)?  
A. : https://specflow.org/contributer-package/
https://specflow.org/blog/hacktoberfest-2020-starts-soon/

Q. What kind of swag (clothing, stickers, ...)?  
A. : stickers

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by tealtail**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Fixes #716  

Adds SpecFlow swag opportunity to data

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -566,6 +566,15 @@
     "dateAdded": "2018-10-23T10:00:46.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "SpecFlow",
+    "difficulty": "easy",
+    "description": "A collection of SpecFlow stickers for getting a PR merged into the SpecFlow repo",
+    "reference": "https://specflow.org/contributer-package",
+    "image": "https://specflow.org/wp-content/uploads/2019/03/Conference_package_specflow_stickers.png",
```
Comment (aslafy-z) [inline data.json:574]:
```suggestion
    "image": "https://i.imgur.com/o7utTj2.png",
```

Comment (tealtail) [reply data.json:574]: 👍

**Review by vikaspotluri123 — APPROVED** [review]

**Comment by vikaspotluri123** [comment]
@aslafy-z oops, I thought you approved!

**Comment by aslafy-z** [comment]
No problem! 🚀

**Pull request merged**